tracks/create-reducer.es6

```diff
@@ -68,6 +68,7 @@ export default function createReducer({END, JUMP, LOAD, PAUSE, PLAY, STOP, TICK}
 
         nextTrack = {
           ...state[id],
+          didEnd: true,
```

Comment:
I don't know about this one as the fact that it stopped doesn't necessarily mean it finished playing the full track which is the intended meaning of `didEnd`. Thoughts @dmcaodha?